src/styles/header.scss

```diff
@@ -0,0 +1,101 @@
+@import './variables';
+@import './mixins';
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  transition: all 0.3s ease-in-out;
+  scroll-behavior: smooth;
+
+  &__link {
+    width: 40px;
+    height: 40px;
+    margin: 10px 50px;
+  }
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__nav {
+    display: none;
+    margin: 0 50px 0 0;
+
+    @include on-tablet {
+      display: flex;
+    }
+  }
+
+  .nav {
+    .header__nav {
```

Comment:
Avoid over-specifying your element, the element's style should not be conditioned by its relative position to other elements/blocks [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Increasing%20an%20Element%20Specificity)